ci/travis/build-manylinux.sh

```diff
@@ -5,6 +5,10 @@ docker_tag="parallelssh/ssh2-manylinux"
 rm -rf build ssh2/libssh2.*
 python ci/appveyor/fix_version.py .
 
+if [[ `uname -m` == "aarch64" ]]; then
```

Comment:
This will introduce unstaged changes and make the repository dirty. That then breaks the python package versioning which relies on repository revision.